workflow/rules/bwa_mem.smk

```diff
@@ -0,0 +1,51 @@
+# vim: syntax=python tabstop=4 expandtab
```

Comment:
I would remove the `vim`-header as people might use different editors.

Reply:
Would it be a problem to leave it? I use vim a lot

Reply:
Me too, but with the snakemake syntaxing I do not need any headers. In my opinion we should separate concerns and remove anything that is not part of the code base. But if it impairs your work heavily we can keep it. 😉